packages/apps/tools/src/utils/network.ts

```diff
@@ -0,0 +1,79 @@
+import type { DefinedNetwork, Network } from '@/constants/kadena';
+import { kadenaConstants } from '@/constants/kadena';
+
+interface IApiHostData {
+  networkId: string;
+  chainId: string;
+}
+
+export interface INetworkDto {
+  networkId: Network;
+  label: string;
+  API: string;
+  apiHost: ({ networkId, chainId }: IApiHostData) => string;
+  estatsHost: (account: string) => string;
+}
+
+const isNetwork = (x: any): x is Network =>
+  ['mainnet01', 'testnet04'].includes(x);
+
+export const getConfigNetworkNames = (): DefinedNetwork[] => {
+  return Object.keys(kadenaConstants).filter((key) =>
+    isNetwork(key),
+  ) as DefinedNetwork[];
+};
+
+export const getAllNetworks = (
+  localStorageNetworks: INetworkDto[],
+): INetworkDto[] => {
+  const allNetworkObjects: INetworkDto[] = [];
+  const configNetworks = getConfigNetworkNames();
+
+  const configNetworksAdded =
+    Boolean(localStorageNetworks) &&
+    localStorageNetworks.find(
+      (item) => item.label === 'Mainnet' || item.label === 'Testnet',
```

Comment:
Maybe we can export the `label`s from the kadena constants file and compare it with that (to make it somewhat more resilient)?